lib/dictBuilder/zdict.h

```diff
@@ -64,6 +64,7 @@ ZDICTLIB_API size_t ZDICT_trainFromBuffer(void* dictBuffer, size_t dictBufferCap
 
 /*======   Helper functions   ======*/
 ZDICTLIB_API unsigned ZDICT_getDictID(const void* dictBuffer, size_t dictSize);  /**< extracts dictID; @return zero if error (not a valid dictionary) */
+ZDICTLIB_API size_t ZDICT_getDictHeaderSize(const void* dictBuffer, size_t dictSize);  /* returns dict header size; returns zero if error (not a valid dictionary or mem alloc failure) */
```

Comment:
Opened question :

For consistency with other functions returning a `size_t`, 
should we rather use 
an error code, compatible with `ZDICT_isError()`, 
instead of `0`
when an error happens ?

cc @terrelln

Reply:
We should definitely return an error code when an error happens.